index.js

```diff
@@ -167,26 +177,13 @@ function fireChangeEvent(target: ImageCropElement, result: {x: number, y: number
   target.dispatchEvent(new CustomEvent('image-crop-change', {bubbles: true, detail: result}))
 }
 
-export class ImageCropElement extends HTMLElement {
```

Comment:
Is the a reason the class is no longer exported?

Reply:
It's exported as `default` at the end of this file so I removed this named export.